<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerClass</class>
 <widget class="QMainWindow" name="ImageViewerClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Nastavenia</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="mode">
         <item>
          <property name="text">
           <string>Polygon/úsečka</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Krivka</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>DDA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bresenham</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="poly">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="polygon">
              <property name="text">
               <string>Polygon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="color">
              <property name="text">
               <string>Zmena farby hrán</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="farba">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="vypln">
              <property name="text">
               <string>Zmena farby výplne</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="farba2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="transform">
         <property name="title">
          <string>Transformácie</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="rot">
            <property name="text">
             <string>Rotácia</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="sca">
            <property name="text">
             <string>Škálovanie</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QDoubleSpinBox" name="sca_x">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sca_y">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="sko">
            <property name="text">
             <string>Skosenie</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="sym">
            <property name="text">
             <string>Symetria</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="sko_spin">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-20.000000000000000</double>
            </property>
            <property name="maximum">
             <double>20.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="rot_spin">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fill">
         <property name="title">
          <string>Vyplnenie trojuholníka</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>jedna farba</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>najbližší sused</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>barycentrická interpolácia</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="krivky">
         <property name="title">
          <string>Typ krivky</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="krivka">
            <item>
             <property name="text">
              <string>hermit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bezier</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>coons</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="kresli">
              <property name="text">
               <string>Draw</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Clear2">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="hermit">
            <property name="title">
             <string>Nastavenie hermitovskej krivky</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Vektor k bodu</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="bod"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Uhol</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="uhol">
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="minimum">
                  <double>-360.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionRename"/>
    <addaction name="actionClear"/>
    <addaction name="actionSet_background_color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSet_background_color">
   <property name="text">
    <string>Set background color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageViewer.qrc"/>
 </resources>
 <connections/>
</ui>
